packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1281,6 +1281,53 @@ void main() {
     await checkDragHandleAndColors();
   });
 
+  testWidgets('Drag handle interactive area size at given dragHandleSize', (WidgetTester tester) async {
+    const Color defaultColor=Colors.blue;
+    const Color hoveringColor=Colors.green;
+
+    Widget buildScaffold(GlobalKey scaffoldKey, {Size? dragHandleSize}) {
+      return MaterialApp(
+        theme: ThemeData.light().copyWith(
+          bottomSheetTheme:  BottomSheetThemeData(
+            dragHandleColor: MaterialStateColor.resolveWith((Set<MaterialState> states) {
```

Comment:
Can `dragHandleColor` be removed so that only the properties relevant to the test are shown?